automatic/blender/update.ps1

```diff
@@ -38,29 +35,32 @@ function global:au_GetLatest {
     $version64 = $version64 -replace $letter,".$num"
   }
 
+  $majorVersion64 = $version64 -split '\.' | Select-Object -First 2
+  $majorVersion64 = $majorVersion64 -join '.'
+
   @{
-    URL64 = Get-ActualUrl $url64
+    URL64 = Get-ActualUrl $version64 $majorVersion64
+    ChecksumType64 = "sha256"
     Version = $version64
+    MajorVersion = $majorVersion64
   }
 }
 
 function Get-ActualUrl() {
-  param([string]$url)
+  param([string]$version, [string]$majorVersion)
 
-  $request = [System.Net.WebRequest]::Create($url)
+  return "https://download.blender.org/release/Blender$majorVersion/blender-$version-windows-x64.msi"
```

Comment:
Usually, we prefer to not have the URL format hardcoded when possible, any reason why you had to change to using a hardcoded format instead of acquiring the actual URL using the old method?

Reply:
See answer below